app/views/admin/outgoing_messages/snippets/edit.html.erb

```diff
@@ -12,5 +12,10 @@
         </div>
       <% end %>
     </div>
+
+    <%= form_tag admin_snippet_path(@snippet), class: 'form form-inline', method: 'delete' do %>
+      <%= submit_tag "Destroy #{@snippet.name}", class: 'btn btn-danger' %>
```

Comment:
`name` is likely going to be far too long.

![Screenshot 2021-02-23 at 09 25 22](https://user-images.githubusercontent.com/282788/108823986-1401e480-75b9-11eb-9464-58351cad8892.png)

Should either truncate or just use `'Destroy snippet'`.